user-manual/preservation/preservation-planning.rst

```diff
@@ -829,9 +829,66 @@ Validation tools
 ^^^^^^^^^^^^^^^^
 
 Archivematica includes two validation tools: `JHOVE`_ and `MediaConch`_.
+
+
+MediaConch
+^^^^^^^^^^
+
 MediaConch was introduced in Archivematica 1.7 as a validation tool specifically
-for Matroska (MKV) files, opening up the possibility to include more specialized
-tools in future releases.
+for files utilizing the Matroska (MKV) container format, the FFV1 video codec
+format, and the LPCM audio codec format. This is the file format used and 
+recommended by Archivematica when performing normalization on audiovisual files.
+For more information about normalization rules, see the :ref:`Normalization
+<normalization>` section below.
+
+In addition to validating files against their format specification, MediaConch 
+can also be used to validate file formats against an internal policy created by 
+any institution. 
+
+Users can use `MediaConchOnline`_ or a local installation of MediaConch to 
+create a policy to check against files being ingested in Archivematica. Policies 
+can include rules such as aspect ratios, bit rate, track information, etc. 
+Policy checking is not restricted to mkv files. A policy can be created for any 
+format which can be characterized by `MediaInfo`_. 
+
+To create a policy within Archivematica:
+
+1. Ensure that the Archivematica processing configuration is set to Perform 
+Policy Checks on originals. You can also use these policies to check 
+preservation and access derivatives created by Archivematica or through manual 
+normalization against a local policy. 
+
+2. In the Preservation Planning tab, click on Commands under Validation. You 
+should see a sample policy called Check against policy 
+PLACEHOLDER_FOR_POLICY_FILE_NAME using MediaConch. Either replace this command 
+or create a new one using it as a template.
+
+3. To use your own command, add the MediaConch-created xml between 
+`POLICY = """` and `""".strip()` in
+the Command section. As an example, here's a very simple policy to check that a 
+file is an mp4:
+
+.. code-block:: none
+
+   <policy type="and" name="Is Mp4?" license="CC-BY-SA-4.0+">
+     <description>New Is Mp4</description>
+     <rule name="Is Mp4?" value="Format" tracktype="General" occurrence="*" operator="=">mp4</rule>
+   </policy>
```

Comment:
I wonder if in this example it would be helpful to include the surrounding text as well? My concern is that someone might not read thoroughly and just include the policy text, erasing the code around it.